src/dawn/pipeline_info.h

```diff
@@ -44,6 +44,8 @@ struct RenderPipelineInfo {
 
   // The framebuffer color render target.  This resides on the GPU.
   ::dawn::Texture fb_texture;
+  // The framebuffer depth and stencil target.  This resides on the GPU.
```

Comment:
I would remove "framebuffer" here.   I normally think of the framebuffer as a colour attachment.

Reply:
removed